app/src/components/authentication/SignUpForm.tsx

```diff
@@ -31,14 +31,13 @@ export const SignUpForm = ({
   errorMessage,
   setErrorMessage,
 }: SignUpFormProps) => {
-  const {handleSubmit, handleChange, values, touched, errors, isValid} =
+  const {handleSubmit, handleChange, values, touched, errors, isValid, dirty} =
```

Comment:
nit but also not familiar with `formik`, what's the difference between `isValid` and `dirty`? are they both related to invalid form input?

Reply:
isValid returns true if there are no errors, and dirty means that the user has touched the field aka it is not deeply equal to the initial values

Reply:
Yep! Mira summed it up. I was running into an issue with the disabled submit button and found [this](https://formik.org/docs/migrating-v2#isvalid) in the Formik docs.